src/operator/rnn.cc

```diff
@@ -138,6 +138,21 @@ static bool RNNType(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+inline static bool RNNStorageType(const nnvm::NodeAttrs& attrs,
+                                  const int dev_mask,
+                                  DispatchMode* dispatch_mode,
+                                  std::vector<int> *in_attrs,
+                                  std::vector<int> *out_attrs) {
+  DispatchMode wanted_mode = DispatchMode::kFCompute;
+
+  #if MXNET_USE_MKLDNN_RNN
```

Comment:
Why not "USE_MKLDNN"? Is there any conflict?

Reply:
If using USE_MKLDNN here, several GPU case in ci will fail

Reply:
What's the error message? Seems we need to fix this issue.

Reply:
many error messages like src/operator/./rnn-inl.h(1328): error: argument of type "mshadow::Stream<mshadow::cpu> *" is incompatible with parameter of type "mshadow::Stream<mshadow::gpu> *"